hr_payroll_extended/__openerp__.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+#  File: __openerp__.py
+#  Module: hr_payroll_extended
+#
+#  Created by sge@open-net.ch
+#
+#  Copyright (c) 2014-TODAY Open-Net Ltd. <http://www.open-net.ch>
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-TODAY OpenERP S.A. <http://www.openerp.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+
+{
+    'name': 'Extended - Payroll',
+    'summary': 'Payroll Extended',
+    'category': 'Human Resources',
+    'author': "Open-Net Sàrl,Odoo Community Association (OCA)",
+    'depends': [
+        'decimal_precision',
+        'hr_payroll',
+        'hr_payroll_account',
+        'hr_contract',
+        'hr_attendance',
+        'account'
```

Comment:
IMO no need to explicitly refer to account, hr_payroll, decimal_precision and hr_contract